bitmovin/package_information.py

```diff
@@ -1,2 +1,2 @@
 NAME = 'bitmovin-python'
-VERSION = '1.6.0'
+VERSION = '1.7.0dev1'
```

Comment:
yes, will be one without dev as soon it's released. 
https://www.python.org/dev/peps/pep-0440/#developmental-releases